### PR TITLE
Create registration done form with the option of updating date of birth 

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

File: molo/profiles/forms.py [hunk 1]
```diff
@@ -54,6 +48,14 @@ def clean_username(self):
         return self.cleaned_data['username']
 
 
+class DateOfBirth(forms.Form):
```
Comment (miltontony) [inline molo/profiles/forms.py]: Call it "DateOfBirthForm"

File: molo/profiles/views.py [hunk 2]
```diff
@@ -21,14 +21,28 @@ def register(request):
                 username=form.cleaned_data['username'],
                 password=form.cleaned_data['password'],
             )
-            user.profile.date_of_birth = form.cleaned_data['date_of_birth']
             user.profile.save()
-            return HttpResponseRedirect(form.cleaned_data.get('next', '/'))
+            return HttpResponseRedirect(form.cleaned_data.get(
```
Comment (miltontony) [inline molo/profiles/views.py]: This should be `return HttpResponseRedirect(reverse('molo.profiles:registration_done'))`

File: molo/profiles/views.py [hunk 3]
```diff
@@ -21,14 +21,28 @@ def register(request):
...
+                                        'molo.profiles:registration_done'))
         return render(request, 'profiles/register.html', {'form': form})
     form = RegistrationForm()
     return render(request, 'profiles/register.html', {'form': form})
 
 
+@csrf_protect
+def register_done(request):
```
Comment (miltontony) [inline molo/profiles/views.py]: This is the old way of doing Form handling. The newer way of doing this is using [Class based views](https://github.com/praekelt/molo.profiles/blob/feature/issue-9-create-registration-done-form-with-DOB/molo/profiles/views.py#L59-L71)

**Comment by smarzban** [comment]
@miltontony ready for review :)

**Comment by smarzban** [comment]
@miltontony ready for review

**Comment by miltontony** [comment]
:+1:

**Pull request merged**